### DISCUSSION TITLE
Issue with ethernet adapter on version 12.3.1

### DISCUSSION THREAD

**Discussion by Pietjuhbel** [arendst/Tasmota#17623] [Q&A]
I don't know if anyone else has faced this same issue, I am not completely sure if this is a Tasmota issue or just me being stupid.

Last week I flashed a M5 Stack EsPoe unit with the latest stable Tasmota. After installation I was unable to get the ethernet adapter working no matter what I tried. 
Since I already have a same unit in production with non of these issues I was stumbled why this one wouldn't.

The ethernet adapter appears completely dead, as in no lights blinking at all.
From experience with the other unit I know that providing Eth_power on the right gpo(5 in this case) is enough to get at least the lights of the adapter blinking. On this one this was not the case, whatever I tried the adapter stayed dead. 

At this point I thought the unit probably had a problem, in a last ditch attempt I flashed the unit with version 12.0.1 same as the working unit and voila everything started working as one would expect. 

Question here, what's the difference between these two versions that could cause this unexpected behavior , or else what am I missing...

Ethernet adapter is of type 1 ETH_PHY_TLK110/ETH_PHY_IP101

**Comment by s-hadinger** [comment]
Can you share logs please? Especially from the boot phase

**Reply by Pietjuhbel** [reply]
Yes I can do that, just have to install latest firmware back onto the device. 
I will get back on this as soon as possible.
